api.swagger.yaml

```diff
@@ -359,6 +443,31 @@ paths:
             schema:
               $ref: '#/components/schemas/user-schema'
         required: true
+  '/users/{user_id}/activity':
+    get:
+      tags:
+        - users
+      description: Return the user's activity feed.
+      parameters:
```

Comment:
I think this needs a pager, or if not we should at least define how many items will be returned

Reply:
added